packages/bpk-component-button/src/BpkButtonBase.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import React from 'react';
+
+import { type Props, propTypes, defaultProps } from './common-types';
+import COMMON_STYLES from './common.scss';
+
+// This was originally depended upon from the bpk-react-utils package, however
+// we decided to inline it in this particular component so as not to bloat the
+// the bundles of consumers who are not yet on webpack 2
+// We'll revisit this again soon.
+const cssModules = (styles = {}) => className =>
```

Comment:
We now have this function in a lot of places. We might not want to rely on `bpk-react-utils`, but we should still dedupe it within in `bpk-component-button`

Reply:
Do we still have a lot of people using webpack 2? Seems like this would not be an issue anymore. 
In any case I don't think we should do this in this PR too, but deduping inside `bpk-component-button` sounds good

Reply:
Yeah agreed we should probably explore moving it to `bpk-react-utils` later